### DISCUSSION TITLE
openjdk17 JAVA_HOME env var is not persistant

### DISCUSSION THREAD

**Discussion by ms4720** [ScoopInstaller/Scoop#5632] [Help and Questions]
## Bug Report

openjdk17 is not setting JAVA_HOME persistently

### Current Behaviour

- install package 
- JAVA_HOME works in the terminal window I installed the package in
- open new terminal window
- JAVA_HOME is not set

### Expected Behaviour

- install package 
- JAVA_HOME works in the terminal window I installed the package in
- open new terminal window
- JAVA_HOME is working in other terminals and across reboots

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:**  5.1.19041.3031

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "alias":  {

              },
    "aria2-enable":  false,
    "aria2-enabled":  false,
    "last_update":  "2023-08-16T10:37:50.6661416-04:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/lukesampson/scoop"
}
```

**Comment by HUMORCE** [comment]
![Screenshot_230901100511](https://github.com/ScoopInstaller/Scoop/assets/4353480/9848eb94-8d50-4943-bb5b-695889b1c70f)

Cannot reproduce.
